client/e2e/pages/onboarding/home/page.spec.ts

```diff
@@ -25,101 +32,27 @@ test.beforeAll(async () => {
-      values: (string | number | boolean)[];
-    } = {
-      text: `
-    INSERT INTO erc.operator (id, status, legal_name, trade_name, cra_business_number, bc_corporate_registry_number, business_structure_id, is_new)
-    VALUES ($1, $2, $3, $4, $5, $6, $7, $8)
-    ON CONFLICT (id)
-    DO UPDATE SET status = EXCLUDED.status;
-  `,
-      values: [
-        2,
-        "Approved",
-        "Existing Operator 2 Legal Name",
-        "Existing Operator 2 Trade Name",
-        "987654321",
-        "def1234567",
-        "BC Corporation",
-        false,
-      ],
-    };
+    let query = upsertOperator;
     // ▶️ Execute the query
     await pool.query(query);
-
     // Upsert a User record
-    query = {
-      text: `
-     INSERT INTO erc.user (user_guid, app_role_id, first_name, last_name , position_title , email, phone_number, business_guid, bceid_business_name )
-    VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9)
-    ON CONFLICT (user_guid)
-    DO UPDATE SET app_role_id = EXCLUDED.app_role_id
-  `,
-      values: [
-        process.env.E2E_INDUSTRY_USER_ADMIN_GUID as string,
-        "industry_user",
-        "Cas",
-        "Bcgov",
-        "ADMINISTRATOR",
-        "email@email.com",
-        "+16044015432",
-        "efb76d57-88b7-4eb6-9f26-ec12b49c14c1",
-        "bceid_business_name",
-      ],
-    };
+    query = upsertIOUserAdmin;
     // ▶️ Execute the query
     await pool.query(query);
-
     // Upsert an User Operator record
-    query = {
-      text: `
-    INSERT INTO erc.user_operator (user_id, role, status, operator_id)
-    VALUES ($1,  $2, $3, $4)
-    ON CONFLICT (user_id, operator_id)
-    DO UPDATE SET role = EXCLUDED.role, status = EXCLUDED.status;
-  `,
-      values: [
-        process.env.E2E_INDUSTRY_USER_ADMIN_GUID as string,
-        "admin",
-        "Approved",
-        2,
-      ],
-    };
+    query = upsertOperatorUser;
     // ▶️ Execute the query
     await pool.query(query);
 
     // 👤 industry_user: bc-cas-dev-secondary
     // Upsert a User record
-    query = {
-      text: `
-         INSERT INTO erc.user (user_guid, app_role_id, first_name, last_name , position_title , email, phone_number, business_guid, bceid_business_name )
-    VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9)
-    ON CONFLICT (user_guid)
-    DO UPDATE SET app_role_id = EXCLUDED.app_role_id
-  `,
-      values: [
-        process.env.E2E_INDUSTRY_USER_GUID as string,
-        "industry_user",
-        "Cas SECONDARY",
-        "Bcgov",
-        "ADMINISTRATOR",
-        "email@email.com",
-        "+16044015432",
-        "efb76d57-88b7-4eb6-9f26-ec12b49c14c1",
-        "bceid_business_name",
-      ],
-    };
+    query = upsertIOUser;
     // ▶️ Execute the query
     await pool.query(query);
 
     // 👤 new user: bc-cas-dev-three
     // Delete User record
-    query = {
-      text: "DELETE FROM erc.user WHERE user_guid = $1",
-      values: [process.env.NEW_USER_GUID as string],
-    };
+    query = deleteNewUser;
```

Comment:
Same question as above, if we don't need it, can we skip adding it in the first place instead of adding it and then deleting it?